simapp/simd/cmd/root.go

```diff
@@ -39,12 +42,17 @@ import (
 // main function.
 func NewRootCmd() *cobra.Command {
 	// we "pre"-instantiate the application for getting the injected/configured encoding configuration
```

Comment:
```suggestion
```

We weren't using depinject / `simapp.AppConfig` here so it we were ensuring to properly use app v1 and app v2.

Reply:
Moved to app-v1 and app-v2.